tasks/prelim.yml

```diff
@@ -190,3 +190,28 @@
       - RHEL-07-021600
       - RHEL-07-021610
       - RHEL-07-021620
+
+- name: "PRELIM | RHEL-07-010480 RHEL-07-010490 | Install python-passlib for hash creation"
+  yum:
+      name: python-passlib
+  when:
+      - inventory_hostname == 'localhost' or inventory_hostname == '127.0.0.1'
+      - rhel_07_020030 or rhel_07_010480
+
+- name: "PRELIM | RHEL-07-020210 RHEL-07-020220| Install SELinux related dependencies"
+  yum:
+      name: "{{ item }}"
+  with_items:
+      - selinux-policy
+      - libselinux-python
+      - selinux-policy-targeted
+      - selinux-policy-mls
```

Comment:
does it work to install both selinux-policy-targeted and selinux-policy-mls?

Reply:
yep, it works, you still have to set the policy explicitly independent of the package being installed/available IIRC